kolibri/plugins/user/assets/src/actions.js

```diff
@@ -22,6 +23,60 @@ function showRoot(store) {
   });
 }
 
+function editProfile(store, edits, session) {
+  // payload needs username, fullname, and facility
+  // used to save changes to API
+  const savedUserModel = FacilityUserResource.getModel(session.user_id);
+  const changedValues = {};
+
+  // TODO set up core session updates
```

Comment:
one quick hack would be to force a refresh after editing. This is similar to what we do on log-out